src/models/StakeRequestRepository.ts

```diff
@@ -72,39 +78,48 @@ export class StakeRequestRepository {
         },
         messageHash: {
           type: DataTypes.STRING,
+          allowNull: true,
+          unique: true,
         },
         amount: {
           type: DataTypes.INTEGER,
+          allowNull: false,
           validate: {
             min: 0,
           },
         },
         beneficiary: {
           type: DataTypes.STRING,
+          allowNull: false,
         },
         gasPrice: {
           type: DataTypes.INTEGER,
+          allowNull: false,
```

Comment:
Thanks for adding this here 👍 
We missed checking that the datatype here is INTEGER and it should be BIGINT.